jazzmin/utils.py

```diff
@@ -198,3 +200,10 @@ def make_menu(user, links, options, allow_appmenus=True):
             )
 
     return menu
+
+
+def has_fieldsets_check(adminform: AdminForm) -> bool:
+    fieldsets = adminform.fieldsets
+    if len(fieldsets) == 1 and fieldsets[0][0] is None:
```

Comment:
mm I guess it can't be zero?

Reply:
Nope, djangos get_fieldsets method seems to always return 1 fieldset, unless you override get_fieldsets and return an empty list..

Reply:
Yeah, it makes sense. I wonder if Django is clever enough to not allow fieldsets to be zero.. probably not 😄 

Otherwise, all looking gooood!

Reply:
I've updated it to bool check fieldsets, shoulda woulda coulda written tests